docs/laravel-integration.md

```diff
@@ -90,140 +110,135 @@ $response = tap($kernel->handle(
-use OpenTelemetry\SDK\Trace\SamplingResult;
-use OpenTelemetry\SDK\Trace\SpanProcessor\BatchSpanProcessor;
+use OpenTelemetry\SDK\Trace\SpanProcessor\SimpleSpanProcessor;
 use OpenTelemetry\SDK\Trace\TracerProvider;
-use OpenTelemetry\API\Trace as API;
 use GuzzleHttp\Client;
 use GuzzleHttp\Psr7\HttpFactory;
 ```
 
 Remember that these imports should go side by side with the default class imports that come with the `index.php` file.
 
-Next, we create a sample recording trace using the [AlwaysOnSampler](https://github.com/open-telemetry/opentelemetry-php/blob/main/sdk/Trace/Sampler/AlwaysOnSampler.php) class, just before the app instance is created like below:
+Next, we create a sample recording trace using
+the [AlwaysOnSampler](https://github.com/open-telemetry/opentelemetry-php/blob/main/sdk/Trace/Sampler/AlwaysOnSampler.php)
+class, just before the app instance is created like below:
 
-```php
-$sampler = new AlwaysOnSampler();
-$samplingResult = $sampler->shouldSample(
-    new Context(),
-    md5((string) microtime(true)),
-    'io.opentelemetry.example',
-    API\SpanKind::KIND_INTERNAL
-);
-```
-
-Since we are looking to export traces to both Zipkin and Jaeger we have to make use of their  exporters;
+Since we are looking to export traces to both Zipkin and Jaeger, we configure a tracer with exporters for both the
+services;
 
 ```php
-$jaegerExporter = new JaegerExporter(
-    'Hello World Web Server Jaeger',
-    'http://localhost:9412/api/v2/spans',
-    new Client(),
-    new HttpFactory(),
-    new HttpFactory()
-);
-
-$zipkinExporter = new ZipkinExporter(
-    'Hello World Web Server Zipkin',
-    'http://localhost:9411/api/v2/spans',
-    new Client(),
-    new HttpFactory(),
-    new HttpFactory()
-);
+$httpClient = new Client();
+$httpFactory = new HttpFactory();
+
+$tracer = (new TracerProvider(
+    [
+        new SimpleSpanProcessor(
+            new OpenTelemetry\Contrib\Jaeger\Exporter(
+                'Hello World Web Server Jaeger',
+                'http://localhost:9412/api/v2/spans',
+                $httpClient,
+                $httpFactory,
+                $httpFactory,
+            ),
+        ),
+        new SimpleSpanProcessor(
+            new OpenTelemetry\Contrib\Zipkin\Exporter(
+                'Hello World Web Server Zipkin',
+                'http://localhost:9411/api/v2/spans',
+                $httpClient,
+                $httpFactory,
+                $httpFactory,
+            ),
+        ),
+    ],
+    new AlwaysOnSampler(),
+))->getTracer('Hello World Laravel Web Server');
 ```
 
-Next, we create a trace then add processors for each trace(One for Jaeger and another for Zipkin). Then we proceed to start and activate a span for each trace. We create a trace only if the RECORD AND SAMPLED sampling result condition passes as follows;
+Next we create a span from our tracer. We don't need to bother with sampling here, since it is handled by the sampler
```

Comment:
I don't think the second sentence about sampling is required.

Reply:
Yes, I included it in light of the content it was replacing. I'll remove it.